mne/io/kit/tests/test_kit.py

```diff
@@ -50,6 +51,7 @@ def test_data():
                               hsp=hsp_path, stim=list(range(167, 159, -1)),
                               slope='+', stimthresh=1)
     assert_true('RawKIT' in repr(raw_py))
+    assert_true(raw_mrk.info['kit_system_id'] == 34)
```

Comment:
@Eric89GXL Because we don't have the proper constants stored yet.... (see https://github.com/mne-tools/mne-python/blob/master/mne/io/kit/constants.py)

`KIT_NY` is itself a Bunch of constants that apply to thee NYU-NY system, and `{system_id: KIT_CONSTANTS}` is a many-to-one mapping since some system ID changes don't lead to changes in any constants, i.e. `KIT_CONSTANTS[34] is KIT_CONSTANTS[33] is KIT_NY`...

Considering the next use case, deciding based on the ID what layout to use, it also seems strange to check `if kit_system_id is KIY_NY` and would be better to have something like 

```
KIT_SYSTEM_NYU_2008 = 32
KIT_SYSTEM_NYU_2009 = 33
...

KIT_LAYOUT = {KIT_SYSTEM_NYU_2008: 'KIT-NY',
              KIT_SYSTEM_NYU_2009: 'KIT-NY',
...
```
?